src/commands/react_native/xcode.rs

```diff
@@ -556,9 +561,7 @@ pub fn wrap_call() -> Result<()> {
             if let (Ok(packager_sourcemap), Ok(mut hermes_sourcemap)) =
                 (packager_sourcemap_result, hermes_sourcemap_result)
             {
-                if hermes_sourcemap.get("debugId").is_none()
-                    && hermes_sourcemap.get("debug_id").is_none()
-                {
+                if !hermes_sourcemap.contains_key("debug_id") {
```

Comment:
Didn't you lose the check for `"debugId"` here?

Reply:
Yes, I did, my mistake. These looked like the same thing

Reply:
#2058 fixes this